### ISSUE TITLE
@Mention function in comment field doesn't show all people

### ISSUE THREAD

**Issue by valery78** [part-up/part-up#1705]
Ik zoek naar Paul Verkuylen, typ hiertoe @Paul en hij toont Paul Verkuylen niet. Hij toont het wel als ik @Paul V typ.

**Comment by noelheesen** [comment]
Dit komt omdat alleen de eerste 10 resultaten worden getoond.

**Comment by valery78** [comment]
Dacht ik al. Is er een oplossing denkbaar waardoor de gebruiker hier geen last van heeft?

**Comment by noelheesen** [comment]
Yes, we could choose to increase the limit. I recommend a maximum of 15 entries (I'd rather type 1 more character than press the down key 14 times). I'd have to check tablet/mobile views to see what happens if the list grows taller than the window.
